prow/config.yaml

```diff
@@ -116,6 +117,88 @@ presubmits:
         hostPath:
           path: /mnt/disks/ssd0
 
+  istio/mixer:
+  - name: mixer-presubmit
+    context: prow/mixer-presubmit.sh
+    always_run: true
+    rerun_command: "@istio-testing bazel test this"
+    trigger: "((?m)^@istio-testing (bazel )?test this,?(\\s+|$)|(?m)^/test( all| bazel),?(\\s+|$))"
+    branches:
+    - master
+    spec:
+      containers:
+      - image: gcr.io/istio-testing/prowbazel:0.1.7
+        args:
+        - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
+        - "--clean"
+        # Bazel needs privileged mode in order to sandbox builds.
+        securityContext:
+          privileged: true
+        env:
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /etc/service-account/service-account.json
+        volumeMounts:
+        - name: service
+          mountPath: /etc/service-account
+          readOnly: true
+        - name: mixer-codecov-token
+          mountPath: /etc/codecov
+          readOnly: true
+        - name: cache-ssd
+          mountPath: /home/bootstrap/.cache
+        ports:
+        - containerPort: 9999
+          hostPort: 9998
+      volumes:
+      - name: service
+        secret:
+          secretName: service-account
+      - name: mixer-codecov-token
+        secret:
+          secretName: mixer-codecov-token
+      - name: cache-ssd
+        hostPath:
+          path: /mnt/disks/ssd0
+    run_after_success:
+    - name: mixer-e2e-smoketest
+      context: prow/mixer-e2e-smoketest.sh
+      always_run: true
+      rerun_command: "@istio-testing smoke test this"
+      trigger: "((?m)^@istio-testing (smoke )?test this,?(\\s+|$)|(?m)^/test( all| bazel),?(\\s+|$))"
```

Comment:
can make this `/test( all| smoke)` please...also thinking we can move away from `@istio-testing smoke test` style comments in favor of promoting the `/test smoke` style  comments and maybe eventually require the `@istio-testing` ones